utils/check_code_style.py

```diff
@@ -564,10 +564,16 @@ def check_include(sanitized_lines, original_lines, file):
 
 
 if __name__ == '__main__':
+	import sys
```

Comment:
in my opinion(has been some time since my last python script) this should go to the top

Reply:
This is pythonic in that modules can be imported under conditional statements (commonly called a conditional import).

In this case, `sys.argv` is only called from within the main method for when a script executes as a standalone program.  I placed the import here (indented with the function) intentionally.

It could be moved to the top but sys is not used anywhere else here.  If this script gets unit tests then they wouldn't need the sys import.

We could opt to always import, though, I'm not against it.  More commonly conditional imports are used for version compatibility in scripts to support multiple versions of Python.